assets/panel/panel.js

```diff
@@ -35,7 +35,22 @@ DebuggerPanel.prototype = {
       toolboxActions: {
         // Open a link in a new browser tab.
         openLink: this.openLink.bind(this),
-        openWorkerToolbox: this.openWorkerToolbox.bind(this)
+        openWorkerToolbox: this.openWorkerToolbox.bind(this),
+        openElementInInspector: async function(grip) {
+          await this.toolbox.initInspector();
+          const onSelectInspector = this.toolbox.selectTool("inspector");
+          const onGripNodeToFront = this.toolbox.walker.gripToNodeFront(grip);
+          const [
+            front,
+            inspector,
+          ] = await Promise.all([onGripNodeToFront, onSelectInspector]);
+
+          const onInspectorUpdated = inspector.once("inspector-updated");
+          const onNodeFrontSet = this.toolbox.selection
+            .setNodeFront(front, { reason: "debugger" });
+
+          return Promise.all([onNodeFrontSet, onInspectorUpdated]);
+        }.bind(this)
```

Comment:
lets move this to a method in Panel or Toolbox. My preference would be if it were in the Toolbox class because it is not really debugger specific

Reply:
@nchevobbe @captainbrosset Could you have a quick look at this function before I move it to https://searchfox.org/mozilla-central/source/devtools/client/framework/toolbox.js?

Reply:
The code in this function looks good to me. I think this is the right thing to do.
When you do move the code to toolbox.js, can you please ask Alex or Yulia to review the patch? I just want to make sure they take a look at it because they've been changing related code with Fission.